frontend/src/app/api.service.ts

```diff
@@ -89,11 +103,11 @@ export class ApiService {
   /**
    * @description This method makes an http delete call to delete a file
    *
-   * @param FileList files a list of files
+   * @param string file a filename
```

Comment:
This does not match the real param name.

Reply:
done